project4/index.html

```diff
@@ -0,0 +1,182 @@
+                </div>
+                <div class="details">
+                    <h3> Brunch and Networking</h3>
+                </div>
+            </div>
+            <div class="time-slot">
+                <div class="details"> 11 am - 1 pm
+                </div>
+                <div class="details">
+                    <h3>
+                        <a href="#Tamara">Tamara Hadeed </a>
+                    </h3>
+                </div>
+                <p class="details"> Streetwear Brand Creation:
+                    Learn from how to startwith a concept to how to market it in an online store.
+                </p>
+            </div>
+            <div class="time-slot">
+                <div class="details"> 1 pm - 2 pm
+                </div>
+                <div class="details">
+                    <h3>Lunch break</h3>
+                </div>
+            </div>
+
+            <div class="time-slot">
+                <div class="details"> 2 pm - 5 pm
+                </div>
+                <div class="details">
+                    <h3><a href="#michael"> Michael Wong</a></h3>
+                </div>
+                <p class="details"> Workshop: Typography Fundamentals
+                </p>
+            </div>
+            <div class="time-slot">
+                <div class="details"> 5 pm - 7 pm
+                </div>
+                <div class="details">
+                    <h3><a href="#alec">Alec Mendez </a></h3>
+                </div>
+                <p class="details"> How to promote your event in Social Media:
+                    Use of Digital Strategy, user psychology, digital branding.
+                </p>
+            </div>
+            <div class="time-slot">
+                <div class="details"> 7 pm - 8 pm
+                </div>
+                <div class="details">
+                    <h3>Closing After Party</h3>
+                </div>
+            </div>
+
+
+    </section>
+
+    <section class="buy-tickets" id="tickets">
+        <h2> Tickets</h2>
+        <form action="">
+            <fieldset>
+                <p class="ticket-selection">Select your ticket</p>
+                <span> <input id=" ticket-1" type="radio" name="tickets" class="sr-only ticket-input" required>
+                    <label for="ticket-1" class="ticket-button"> Ticket</label> </span>
+                <span> <input id="ticket-2" type="radio" name="tickets" class="sr-only ticket-input" required>
+                    <label for="ticket-2" class="ticket-button"> Student Ticket</label> </span>
+                <span> <input id="ticket-3" type="radio" name="tickets" class="sr-only ticket-input" required>
+                    <label for="ticket-3" class="ticket-button"> Ticket + Drink </label> </span>
+            </fieldset>
+
+            <fieldset class="register">
+                <span> <label for="name">First Name</label>
+                    <input for="name" type="text" required></span>
```

Comment:
```suggestion
                    <input id="name" type="text" required></span>
```

The `for` attribute is used on `label`s to connect them to the `id` of an `input` (or other form element). Wanted to point this out since it was consistently in this form.